news/2023/2023-07-06-introducing-new-beatmap-pack-categories.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "Introducing New Beatmap Pack Categories"
+date: 2023-07-06 09:00:00 +0000
+---
+
+The beatmap pack scene is taking off in 2023 with a number of all-new categories and some new talent to boot!
+
+![](/wiki/shared/news/2023-07-06-introducing-new-beatmap-pack-categories/banner.jpg)
+
+What's the one thing better than a beatmap? *More* beatmaps.
+
+For this reason, the engineers over at the osu! beatmap pack factory have been working hard to scale up operations in order to deliver high-quality packs right to your doorstep, and their new line-up is sure to excite you!
+
+But first, we'd like to welcome some new talent into the beatmap packing business, namely [Shige-Tori\[a\]](https://osu.ppy.sh/users/4459449), who has for a long while catalogued existing packs in an [exhaustively expansive unofficial spreadsheet](https://docs.google.com/spreadsheets/d/1gcXL9gubcWEKY1X2taxJdBGjFrqEpkNmjHU7LFpcJRo/edit#gid=430213160). We'd also like to give a somber and grateful farewell to [Stefan](https://osu.ppy.sh/users/626907), who has kept the lights on at the beatmap pack assembly line for *many, many years* with his tireless toil.
+
+"What are beatmap packs", you ask? They are collections of hand-picked beatmaps that you can download in bulk from one place, instead of tediously looking around to find them all. The [beatmap pack listing](https://osu.ppy.sh/beatmaps/packs) even tracks your completion on each one!
+
+"Why would you complete beatmap packs?" Well, most of them do absolutely nothing (not to discourage any completionists out there), but a *special few* may give you a shiny reward...
+
+Now, let's stop asking questions and take a look at the new beatmap pack categories:
+
+![](/wiki/shared/news/2023-07-06-introducing-new-beatmap-pack-categories/new-beatmap-pack-categories.png)
+
+## Project Loved packs
+
+Project Loved has been bringing some of osu!'s finest creations to your screens since 2016, and the time has come to bring some *HIGH-DENSITY LOVED GOODNESS* to your songs folder. Each pack contains a selection of each season's greatest hits, as chosen by the Loved captains themselves.
+
+Show them some *love* over at the [Project Loved pack listing](https://osu.ppy.sh/beatmaps/packs?type=loved), and perhaps even earn some *shiny goodies* while you're at it!
+
+## Tournament packs
+
+For years people have been asking the question, *"where can I find a nice selection of easy-to-download mappools from past tournaments?"*. Turns out the answer has been found, as tournament packs are now alive and kicking! Enjoy some sensible Quarterfinal pools, or test your limits with a Grand Final pool from any of the most hype tournaments to have graced the game.
+
+For now, the focus is on getting all of the World Cup mappools uploaded, but expect to see other tournaments joining them very soon!
+
+Give yourself a challenge over at the [tournament pack listing](https://osu.ppy.sh/beatmaps/packs?type=tournament) today!
+
+## Centurion packs
```

Comment:
thoughts on also announcing the badge here? they're pretty much confirmed, ready, and will be applied around the same time this newspost is live. i can DM you the badge files

note: you might want to adjust the newspost banner+title for this

Reply:
i'm gonna leave the banner and title as it is, as the main focus is still the packs over the badges.